libmscore/rest.cpp

```diff
@@ -924,30 +940,57 @@ void Rest::localSpatiumChanged(qreal oldValue, qreal newValue)
       }
 
 //---------------------------------------------------------
-//   getProperty
+//   propertyDefault
 //---------------------------------------------------------
 
-QVariant Rest::getProperty(Pid propertyId) const
+QVariant Rest::propertyDefault(Pid propertyId) const
       {
       switch (propertyId) {
             case Pid::GAP:
-                  return _gap;
+                  return false;
+            case Pid::YPOS_MMREST_NUMBER:
+                  return -1.5;
```

Comment:
Default value here should point to the score style.

Reply:
Good catch, thanks